frontend/src/components/NumberInput/NumberInput.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styles from './NumberInput.module.scss'
+
+const NumberInput = ({
+  placeholder,
+  label,
+  value,
+  id,
+  onChange,
+  max,
+  min = 0,
+  step,
+}) => {
+  return (
+    <>
+      <input
+        id={id}
+        placeholder={placeholder}
+        value={value}
+        type="number"
+        onChange={onChange}
+        max={max}
+        min={min}
+        step={step}
+        className={styles.numberInput}
+      ></input>
```

Comment:
Input można od razu zamknąć w jednym tagu. Można na to spróbować dodać regułę eslinta